### ISSUE TITLE
when training yolov5 on custom dataset on spider show this error? how to resolve  this error in pc?

### ISSUE THREAD

**Issue by anambibi** [fizyr/keras-retinanet#1580]
!python C:/Users/Student/Desktop/yolov5-master/yolov5-master/train.py --img 416 --batch 16 --epochs 100 --data C:/Users/Student/Desktop/yolov5-master/yolov5-master --cfg C:/Users/Student/Desktop/yolov5-master/yolov5-master/models/yolov5s.yaml --weights '' --name yolov5s_results  --cache--img 416 --batch 16 --epochs 100 --data C:/Users/Student/Desktop/yolov5-master/yolov5-master --cfg C:/Users/Student/Desktop/yolov5-master/yolov5-master/models/yolov5s.yaml --weights '' --name yolov5s_results  --cache
usage: train.py [-h] [--weights WEIGHTS] [--cfg CFG] [--data DATA] [--hyp HYP]
                [--epochs EPOCHS] [--batch-size BATCH_SIZE] [--imgsz IMGSZ]
                [--rect] [--resume [RESUME]] [--nosave] [--noval]
                [--noautoanchor] [--noplots] [--evolve [EVOLVE]]
                [--bucket BUCKET] [--cache [CACHE]] [--image-weights]
                [--device DEVICE] [--multi-scale] [--single-cls]
                [--optimizer {SGD,Adam,AdamW}] [--sync-bn] [--workers WORKERS]
                [--project PROJECT] [--name NAME] [--exist-ok] [--quad]
                [--cos-lr] [--label-smoothing LABEL_SMOOTHING]
                [--patience PATIENCE] [--freeze FREEZE [FREEZE ...]]
                [--save-period SAVE_PERIOD] [--local_rank LOCAL_RANK]
                [--entity ENTITY] [--upload_dataset [UPLOAD_DATASET]]
                [--bbox_interval BBOX_INTERVAL]
                [--artifact_alias ARTIFACT_ALIAS]
train.py: error: unrecognized arguments: --cache--img 416
